jekyll/_cci2/executor-types.md

```diff
@@ -8,15 +8,35 @@ order: 10
 ---
 [building-docker-images]: {{ site.baseurl }}/2.0/building-docker-images/
 
-This version of CircleCI enables you to use Docker Images instead of using the full Linux OS container. This change increases performance by building only what is required for your application, and it requires selecting an image in your `.circleci/config.yml` file, for example: 
+This version of CircleCI enables you to use Docker Images instead of using the full Linux OS container. This change increases performance by building only what is required for your application. This change requires selecting an image in your `.circleci/config.yml` file that defines the primary container image where all steps run, for example: 
 ```
 jobs:
   build:
     docker:
       - image: buildpack-deps:trusty
 ```
-To make the transition easy, CircleCI maintains convenience images on Docker Hub for popular languages. See [Using Pre-Built CircleCI Docker Images]({{ site.baseurl }}/2.0/circleci-images/) for the complete list of names and tags.    
+In this example, all steps run in this first listed image. To make the transition easy, CircleCI maintains convenience images on Docker Hub for popular languages. See [Using Pre-Built CircleCI Docker Images]({{ site.baseurl }}/2.0/circleci-images/) for the complete list of names and tags.    
```

Comment:
suggest change "In this example, all steps run in this first listed image." to "In this example, all steps run in this first listed container."